src/zmap.1

```diff
@@ -212,8 +219,8 @@ Minimum hitrate that scan can hit before scan is aborted
 Comma\-separated list of cores to pin to
 .
 .TP
-\fB\-\-ignore\-invalid\-hosts\fR
-Ignore invalid hosts in whitelist/blacklist file
+\fB\-\-ignore\-blacklist\-errors\fR
+Ignore invalid entries in whitelist/blacklist file
```

Comment:
This option does seem to make ZMap ignore malformed entries like `123.456./24`. Added clarification in dba82b5223df7e77ce6d338716342b967c12d66a.